plugin/src/main/java/lol/pyr/znpcsplus/npc/NpcRegistryImpl.java

```diff
@@ -57,7 +63,7 @@ private void unregister(NpcEntryImpl entry) {
     }
 
     private void unregisterAll() {
-        for (NpcEntryImpl entry : npcList) entry.getNpc().delete();
+        for (NpcEntryImpl entry : getAllModifiable()) entry.getNpc().delete();
```

Comment:
I think this should use a list of all of the Npcs that are saveable, not modifiable.

Reply:
Will change

Reply:
fixed in d7071ce